### PR TITLE
[matter.js] Added damping to constraints

### PR DESCRIPTION
**Pull request by Rheeseyb**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://brm.io/matter-js/docs/classes/Constraint.html#property_damping
- [x] Increase the version number in the header if appropriate.

### CODE REVIEW

**Review by hasparus — APPROVED** [review]
Thanks for your contribution 👍 

Link to matter code for future reference: https://github.com/liabru/matter-js/commit/44c4d1f

File: types/matter-js/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for Matter.js - 0.10.0
+// Type definitions for Matter.js - 0.10.1
```
Comment (DanielRosenwasser) [inline types/matter-js/index.d.ts]: Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: types/matter-js/index.d.ts (hunk 1 above)
Comment (hasparus) [reply types/matter-js/index.d.ts]: @DanielRosenwasser IMO it's work for another pull request. `matter-js/index.d.ts` has a lot of lint errors (especially jsdoc asterisks alignments lol) :D and there's no fixer (nor prettier support) for them.

https://github.com/hasparus/DefinitelyTyped/pull/1

Comment (DanielRosenwasser) [reply types/matter-js/index.d.ts]: Sorry for the delay; just as a heads up, this probably doesn't publish the version as you'd expect.

**Review by DanielRosenwasser — APPROVED** [review]

File: types/matter-js/index.d.ts (hunk 1 above)
Comment (DanielRosenwasser) [reply types/matter-js/index.d.ts]: I'll merge this in just not to hold you up too much longer :D

**Pull request merged**